pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-gql-pygen"
-version = "0.3.0"
+version = "0.3.1"
```

Seems like a bigger feature change, thus worthy of a feature bump 
```suggestion
version = "0.4.0"
```

---

I realize there is a breaking change in the code, you rename the `dm topython --name` to `dm topython --prefix` thus the minor should be bumped.